app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1934,3 +1913,38 @@ h1.admin-welcome {
   font-size: 24px;
   color: white;
 }
+
+.order-id {
+  text-align: center;
+  margin: 50px 0px 10px 0px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 40px;
+  color: white;
+}
+
+.order-deets {
+  text-align: center;
+  margin: 10px 0px 10px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 0px` instead of `10px 0px 10px 0px`<br>`0px` should be written without units as `0`